manifests/network/contrail/analytics.pp

```diff
@@ -259,18 +259,18 @@
   if $auth_version == 2 {
     $keystone_config_ver = {}
     $auth_url_suffix = 'v2.0'
-    $vnc_authn_url = "/v2.0/tokens"
+    $vnc_authn_url = '/v2.0/tokens'
   } else {
     $keystone_config_ver = {
       'auth_type'               => $keystone_auth_type,
       'project_domain_name'     => $keystone_project_domain_name,
       'user_domain_name'        => $keystone_user_domain_name,
     }
     $auth_url_suffix = 'v3'
-    $vnc_authn_url = "/v3/auth/tokens"
+    $vnc_authn_url = '/v3/auth/tokens'
   }
   if $auth_protocol == 'https' {
-    $auth_url = "${auth_protocol}://${auth_host}:${auth_port_ssl}/${auth_url_suffix}"
+    $auth_url = "${auth_protocol}://${auth_host}:${auth_port}/${auth_url_suffix}"
```

Comment:
Is it really correct change: auth_port_ssl => auth_port   ?

Reply:
I’ve just reverted wrong change, occurred during cherry-pick from another branch.

Reply:
done